server.js

```diff
@@ -1,27 +1,11 @@
-app.get('/slambook', async (req, res) => {
+app.post('/slambook', async (req, res) => {

```

Comment:
It would be helpful to validate the input before creating a new entry to ensure data integrity.